src/app/search/search-filters/freeform.tsx

```diff
@@ -61,28 +66,25 @@ const getUniqueItemNamesFromManifest = _.once(
         // (user's owned items will be included regardless)
         return !powerCap || !irrelevantPowerCaps.has(powerCap);
       })
-      .map((i) => i.displayProperties.name);
-    return _.uniq(itemNames);
+      .map((i) => i.displayProperties.name.toLowerCase());
+    return Array.from(new Set(itemNames));
   }
 );
 
 const nameFilter: FilterDefinition = {
   keywords: 'name',
   description: tl('Filter.PartialMatch'),
   format: 'freeform',
-  // could we do this with a for loop faster,
-  // with wayyyyy more lines of code?? absolutely
   suggestionsGenerator: ({ d2Manifest, allItems }) => {
     if (d2Manifest && allItems) {
       const myItemNames = allItems
         .filter((i) => i.bucket.inWeapons || i.bucket.inArmor || i.bucket.inGeneral)
-        .map((i) => i.name);
+        .map((i) => i.name.toLowerCase());
       // favor items we actually own
-      const allItemNames = getUniqueItemNamesFromManifest(
-        Object.values(d2Manifest.InventoryItem.getAll())
-      );
-      return _.uniq([...myItemNames, ...allItemNames]).map(
-        (s) => `name:${quoteFilterString(s.toLowerCase())}`
+      const allItemNames = getUniqueItemNamesFromManifest(d2Manifest.InventoryItem.getAll());
+      return Array.from(
+        new Set([...myItemNames, ...allItemNames]),
+        (s) => `name:${quoteFilterString(s)}`
```

Comment:
This was the most significant change - turns out _.uniq is nowhere near as efficient as just making a Set for large lists.